controller/linodeobjectstoragebucket_controller_test.go

```diff
@@ -153,7 +153,7 @@ var _ = Describe("LinodeObjectStorageBucket controller", func() {
 	It("should reconcile an object delete", func() {
 		mockClient := mock.NewMockLinodeObjectStorageClient(mockCtrl)
 
-		for i := range 2 {
+		for i := 0; i < 2; i++ {
```

Comment:
Go 1.22 supports `range int` syntax. What version are you running locally?

Reply:
golangci was giving me errors